news/models.py

```diff
@@ -7,3 +7,71 @@ class Subscriber(models.Model):
 
     def __str__(self):
         return self.user.username
+
+
+class Agency(models.Model):
```

Comment:
This should be modeled after a news agency. It probably has a name, description, image, ...